environment.yml

```
@@ -4,25 +4,26 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - artic-porechop==0.3.2pre
-  - artic-tools==0.2.6
-  - bcftools=1.10.2
-  - biopython=1.76
-  - bwa=0.7.17
-  - clint=0.5.1
-  - htslib=1.10.2
-  - longshot=0.4.1
-  - medaka=1.0.3
-  - minimap2=2.17
-  - multiqc
-  - muscle=3.8
-  - nanopolish=0.13.2
-  - nomkl
-  - pandas=0.23.0
+  - artic-porechop =0.3.2pre
+  - artic-tools =0.2.6
+  - longshot =0.4.5
+  - bcftools =1.15.1
+  - biopython =1.79
+  - bwa =0.7.17
+  - clint =0.5.1
+  - htslib =1.15.1
+  - medaka >=1.6.1
+  - minimap2 =2.24
+  - multiqc =1.13
+  - muscle =3.8
+  - nanopolish =0.14.0
+  - pandas =1.4.4
```

Não sei se foi proposital, mas acho melhor deixar sem espaço sempre. 

```suggestion
  - artic-porechop=0.3.2pre
  - artic-tools=0.2.6
  - longshot=0.4.5
  - bcftools=1.15.1
  - biopython=1.79
  - bwa=0.7.17
  - clint=0.5.1
  - htslib=1.15.1
  - medaka>=1.6.1
  - minimap2=2.24
  - multiqc=1.13
  - muscle=3.8
  - nanopolish=0.14.0
  - pandas=1.4.4
```